### DISCUSSION TITLE
Add support for pinnning/unpinning posts

### DISCUSSION THREAD

**Discussion by strk** [cheeaun/phanpy#357] [Ideas]
I've discovered Mastodon supports the concept of "pinned" posts, and Phanpy is already aware of them as it gives them special visibility (that's how I discovered the existence of the concept). But Phanpy does not seem to have the a way for users to toggle the "pin" status of their posts, which was surprising to me.

This thread is meant as a feature request for adding such toggle configuration, unless it's already there and I was not able to see it

**Comment by cheeaun** [comment]
You're right. This feature is currently missing. You can convert this to an issue too.

**Reply by strk** [reply]
I've no idea how to do this, but I saw when filing an issue (which was my first choice) that there was not "Feature request" in the template, but only "Bug" and I'm not sure this can be called a bug :) 

Please convert it to an issue if you think it fits

**Reply by cheeaun** [reply]
Oh, you're right! 🫣 Will look into this.

**Comment by cheeaun** [comment]
Closing, converted to issue here https://github.com/cheeaun/phanpy/issues/359
